quick_tour/the_big_picture.rst

```diff
@@ -155,11 +155,12 @@ because that will be explained in the next section)::
 
     use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
     use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+    use Symfony\Component\HttpFoundation\Request;
```

Comment:
I agree with @xabbuh here. I can see why it's usefull at the code side, I think it's confusion when doing it in the docs.